File: src/java/org/voegtle/weatherstation/server/request/RepairUrlParameter.java
package org.voegtle.weatherstation.server.request;

import org.voegtle.weatherstation.server.util.DateUtil;

import javax.servlet.http.HttpServletRequest;

public class RepairUrlParameter extends UrlParameter {

  public RepairUrlParameter(HttpServletRequest request, DateUtil dateUtil) {
    super(request, dateUtil, null);
  }
}

File: src/main/java/org/voegtle/weatherstation/server/request/RepairUrlParameter.kt
package org.voegtle.weatherstation.server.request

import org.voegtle.weatherstation.server.util.DateUtil

import javax.servlet.http.HttpServletRequest

class RepairUrlParameter(request: HttpServletRequest, dateUtil: DateUtil) : UrlParameter(request, dateUtil,
                                                                                         DataType.UNDEFINED) {
}
